README.md

```diff
@@ -3,6 +3,18 @@
 The intent of this Operator is to deploy common workloads to establish
 a performance baseline of Kubernetes cluster on your provider.
 
+## Installation
+
+Installing the benchmark-operator is easiest by using the helm chart and can be done with the following commands.
+
+> Note: If running on openshift you'll need to run this command before installing the chart. `oc adm policy -n my-ripsaw add-scc-to-user privileged -z benchmark-operator`
+
+```bash
+git clone https://github.com/cloud-bulldozer/benchmark-operator
+cd benchmark-operator/charts/benchmark-operator
+helm install benchmark-operator . -n my-ripsaw --create-namespace
```

Comment:
We might want to mention that helm needs to be installed on the box and may be also include the helm command to delete the operator that you have in the commit message.